.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Objective
+<!--- Describe your changes in detail -->
```

Comment:
This heading and comment seem to say opposite things.

The objective might be "Improve accessibility of button".

Changes in detail might be "Add `tabindex=0` to `div`s that should receive keyboard focus."

If you want both, maybe something like:

> # What and how
> <!-- Describe what this change achieves, and the details of how it works. -->